src/main/java/org/fundacionjala/pivotal/objects/DeleteProject.java

```diff
@@ -0,0 +1,62 @@
+package org.fundacionjala.pivotal.objects;
+
+import org.fundacionjala.pivotal.common.Base;
+import org.fundacionjala.pivotal.common.CommonMethods;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * Class for Deleted Project Page Object.
+ */
+public class DeleteProject extends Base {
+
+    @FindBy(how = How.XPATH, using = "//*[contains(@href,'#') and  text()='Delete']")
+    // //*[contains(@href,'#') and  text()='Delete']
+    private WebElement buttonDeleteProject;
+
+    @FindBy(how = How.ID, using = "confirm_delete")
+    private WebElement buttonConfirmDelete;
+
+    @FindBy(how = How.CSS, using = "#notice")
+    private WebElement messageDelete;
+
+    /**
+     * Method for set Message Delete.
+     * @return messageDeleted is Displayed.
+     */
+    public boolean setMessageDelete() {
+        CommonMethods.waitWebElement(messageDelete);
+        return messageDelete.isDisplayed();
+    }
+
+    /**
+     * Method for set Button Delete Project.
+     */
+    private void setButtonDeleteProject() {
+        CommonMethods.clickWebElement(buttonDeleteProject);
+    }
+
+    /**
+     * Method for Button Confirm delete.
+     */
+    private void setButtonConfirmDelete() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "setButtonConfirmDelete" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)